tests/integration/modules/test_test.py

```diff
@@ -83,3 +89,35 @@ def test_outputter(self):
         test.outputter
         """
         self.assertEqual(self.run_function("test.outputter", ["text"]), "text")
+
+    @slowTest
+    def test_versions_version(self):
+        """
+        test.versions
+        """
+
+        # copied from salt/version.py
```

Comment:
Shouldn't we instead refactor salt/version.py so we can test it, mocking it if necessary?